Library/Formula/voikko-fi.rb

```diff
@@ -0,0 +1,26 @@
+class VoikkoFi < Formula
```

Comment:
Maybe. Voikko supports multiple languages and Finnish dictionaries are not required, but Finnish is the primary language and the reason I originally packaged Voikko for Homebrew.